app/src/main/java/com/x8bit/bitwarden/ui/platform/components/stepper/BitwardenStepper.kt

```diff
@@ -78,9 +77,7 @@ fun BitwardenStepper(
                     }
                 },
                 isEnabled = isIncrementEnabled && !isAtRangeMaximum,
-                modifier = Modifier.semantics {
-                    increaseButtonTestTag?.let { testTag = it }
-                },
+                modifier = Modifier.testTag("IncrementValue"),
```

Comment:
What about the custom test tag?

Reply:
Since we consider the Stepper a component, we require a fixed ID for each inner element (Increment/Decrease buttons, inner text field, etc.).
That's why I made that change. Does it affect your UI unit tests?

Reply:
No, are test should be fine. We never use test tag in test since they make for a weaker test.

But can you remove the `increaseButtonTestTag` and `decreaseButtonTestTag` params if they are no longer needed.

Reply:
Done!

Reply:
You removed the properties but did not update all the places that call those properties

Reply:
My bad! Fixed